### PR TITLE
PR: reverse chronify posts

### PR DESCRIPTION
**Pull request by jkingharman**
…lated feature spec

### CODE REVIEW

File: spec/features/user_can_submit_posts_spec.rb [hunk 1]
```diff
@@ -1,11 +1,22 @@
 require 'rails_helper'
 
 RSpec.feature "Timeline", type: :feature do
+  # now redundant
```
Comment (tbscanlon) [inline spec/features/user_can_submit_posts_spec.rb]: Can we remove the comment?

**Review by tbscanlon — CHANGES_REQUESTED** [review]
ok go

File: spec/features/user_can_submit_posts_spec.rb (hunk 1 above)
Comment (marysalemme) [inline spec/features/user_can_submit_posts_spec.rb]: we should get rid of comments before merging with master 🐶

File: spec/features/user_can_submit_posts_spec.rb [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
-    visit "/posts"
-    click_link "New post"
-    fill_in "Message", with: "Hello, world!"
-    click_button "Submit"
+    create_post
     expect(page).to have_content("Hello, world!")
   end
+
+  scenario "posts have the time" do
+    Timecop.freeze(Time.parse("2017-07-25 14:56:46.490151 +0100"))
```
Comment (marysalemme) [inline spec/features/user_can_submit_posts_spec.rb:10]: we should unfreeze the time after the feature test

**Review by marysalemme — COMMENTED** [review]
We left two comments:
- a commented out line was left in the code
- the time is kept frozen by Timecop in one of the feature tests

Can you please consider our suggestions?
Mary & Miho

**Review by tbscanlon — APPROVED** [review]
:shipit:

**Pull request closed without merging**